src/components/cart/CartItemOrderPreview/CartItemOrderPreview.tsx

```diff
@@ -61,7 +62,13 @@ export const CartItemOrderPreview = ({ videos }: Props) => {
               <span>{video.title}</span>
               {video.price && <PriceBadge price={video.price} />}
             </Typography.Title2>
-            <VideoLicenseDuration video={video} />
+            <FeatureGate feature="BO_WEB_APP_LICENSING_DETAILS">
+              <Badge
+                customClassName={s.licenseRestrictionsInfo}
+                data-qa="video-license-duration-badge"
+                label="This item has restrictions associated with the license."
+              />
```

Comment:
Do we not have any kind of check for restrictions being associated with the video or do all video have restrictions of some kind? If it's the latter I'm not sure I fully understand the need for the story in general as it seems slightly redundant to display this on all videos.

Reply:
I had the same question but there seems to be atleast some kind of restriction on all videos. 
But the point is valid if it makes sense to always show this. Maybe we can start this conversation in the standup today